addons/website_sale/models/res_config.py

```diff
@@ -28,36 +32,64 @@ def _default_order_mail_template(self):
+    module_website_sale_digital = fields.Boolean("Digital Content", help='Sell digital files.')
+    module_sale_stock = fields.Boolean("Delivery Orders")
+    module_account_taxcloud = fields.Boolean("TaxCloud")
     module_portal = fields.Boolean("Activate the customer portal", help="""Give your customers access to their documents.""")
     # the next 2 fields represent sale_pricelist_setting from sale.config.settings, they are split here for the form view, to improve usability
-    sale_pricelist_setting_split_1 = fields.Selection([
-        (0, 'A single sales price per product'),
-        (1, 'Several prices selectable through a drop-down list or applied automatically via Geo-IP'),
-        ], default=0, string="Pricing Strategy")
+    sale_pricelist_setting_split_1 = fields.Boolean(default=0, string="Multiple Prices per Products")
     sale_pricelist_setting_split_2 = fields.Selection([
-        (0, 'Specific prices per customer segment, currency, etc.'),
-        (1, 'Advanced pricing based on formulas (discounts, margins, rounding)')
+        (0, 'Multiple prices per product (e.g. customer segments, currencies)'),
+        (1, 'Prices computed from formulas (discounts, margins, roundings)')
         ], default=0, string="Sales Price",
         help='Specific prices per customer segment, currency, etc.: new pricing table available in product detail form (Sales tab).\n'
              'Advanced pricing based on formulas (discounts, margins, rounding): apply price rules from a new *Pricelists* menu in Configuration.')
     group_sale_pricelist = fields.Boolean("Use pricelists to adapt your price per customers",
         implied_group='product.group_sale_pricelist',
         help="""Allows to manage different prices based on rules per category of customers.
                 Example: 10% for retailers, promotion of 5 EUR on this product, etc.""")
+    group_product_variant = fields.Boolean("Attributes and Variants", implied_group='product.group_sale_pricelist')
     group_pricelist_item = fields.Boolean("Show pricelists to customers",
         implied_group='product.group_pricelist_item')
     group_product_pricelist = fields.Boolean("Show pricelists On Products",
         implied_group='product.group_product_pricelist')
-    order_mail_template = fields.Many2one('mail.template', string='Order Confirmation Email', readonly=True, default=_default_order_mail_template, help="Email sent to customer at the end of the checkout process")
+    order_mail_template = fields.Many2one('mail.template', string='Email Template', readonly=True, default=_default_order_mail_template, help="Email sent to customer at the end of the checkout process")
+    group_show_price_subtotal = fields.Boolean(
+        "Show subtotal",
+        store=True,
+        implied_group='sale.group_show_price_subtotal',
+        group='base.group_portal,base.group_user,base.group_public')
```

Comment:
Why all these groups on the model? 

- Public users don't have access to backend, so I don't think that it's useful to allow them to access this field
- Group portal doesn't (or shouldn't have access to configurations)
- Every user that isn't a public user and a portal user is considered as an employee in Odoo. So every users that should access this field is already a base.group_user